scripts/build/Dockerfile_llvm_build

```diff
@@ -0,0 +1,27 @@
+# Build LLVM
+ARG LLVM_VERSION_SHORT
+ARG REPOSITORY
+FROM ${REPOSITORY}/base
+LABEL maintainer="Martin Nowack <m.nowack@imperial.ac.uk>"
+
+ENV DOCKER_BUILD=1
+
+# Define all variables that can be changed as argument to the docker build
+ARG BASE=/tmp
```

Comment:
The `ARG BASE=/tmp`keeps pooping up, so it would be very annoying to change it. I would say either get rid of it (by explicitly using /tmp or more sensible names) or make it a single point of change.